src/ray/util/logging.h

```diff
@@ -141,8 +141,12 @@ enum class RayLogLevel {
 #define RAY_CHECK_WITH_DISPLAY(condition, display)                                 \
   RAY_PREDICT_TRUE((condition))                                                    \
   ? RAY_IGNORE_EXPR(0)                                                             \
-  : ::ray::Voidify() & (::ray::RayLog(__FILE__, __LINE__, ray::RayLogLevel::FATAL) \
-                        << " Check failed: " display " ")
+  : ::ray::Voidify() &                                                             \
+          (::ray::RayLog(__FILE__, __LINE__, ray::RayLogLevel::FATAL)              \
+           << " Congratulations! You've discovered a bug in Ray that made the "    \
+              "system behave in an unexpected way. Please report this issue at "   \
+              "https://github.com/ray-project/ray/issues and we'll try to fix it " \
+              "ASAP! Check failed: " display " ")
```

Comment:
It's a little too tongue-in-cheek IMO. Would prefer something more straightforward:
```suggestion
  : ::ray::Voidify() &                                                             \
          (::ray::RayLog(__FILE__, __LINE__, ray::RayLogLevel::FATAL)              \
           << " An unexpected system state has occurred. You have likely "      \
              "discovered a bug in Ray. Please report this issue at "   \
              "https://github.com/ray-project/ray/issues and we'll work with you to fix it. " \
              "Check failed: " display " ")
```

Reply:
the user will feel unappreciated for their usage 😞